packages/database/src/migrations/20210705232632-AddECEGoalsDrillDowns-modifies-data.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+import { insertObject, generateId, findSingleRecord, findSingleRecordBySql } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const generateReport = (dataElement, target) => ({
+  fetch: {
+    dataElements: [dataElement],
+    aggregations: [
+      {
+        type: 'MOST_RECENT',
+      },
+      {
+        type: 'SUM_PER_ORG_GROUP',
+        config: {
+          dataSourceEntityType: 'sub_district',
+          aggregationEntityType: 'requested',
+        },
+      },
+    ],
+  },
+  transform: [
+    'keyValueByDataElementName',
+    {
+      transform: 'select',
+      "'Current'": `$row.${dataElement}/100`,
+      "'Target'": `${target || undefined}`,
+      "'timestamp'": 'periodToTimestamp($row.period)',
+    },
+    {
+      transform: 'aggregate',
+      timestamp: 'group',
+      '...': 'last',
+    },
+  ],
+});
+
+const FRONT_END_CONFIG = {
+  type: 'chart',
+  chartType: 'line',
+  xName: 'Year',
+  yName: 'Rate',
+  periodGranularity: 'year',
+  valueType: 'percentage',
+  chartConfig: {
+    Current: {
+      color: '#f44336',
+    },
+    Target: {
+      color: '#4caf50',
+    },
+  },
+};
+
+const CONFIG_0_2_REPORT = {
+  code: 'LESMIS_enrolment_ece_0_2_target',
+  reportConfig: generateReport('er_district_ece_0_2_t', '0.07'),
+  frontEndConfig: { ...FRONT_END_CONFIG, name: 'Enrolment rate of 0-2 year old students in ECE' },
+};
+
+const CONFIG_3_4_REPORT = {
+  code: 'LESMIS_enrolment_ece_3_4_target',
+  reportConfig: generateReport('er_district_ece_3_4_t'),
+  frontEndConfig: {
+    ...FRONT_END_CONFIG,
+    name: 'Enrolment rate of 3-4 year old students in ECE',
+    chartConfig: { Current: { color: '#f44336' } }, // Remove target from legend
+  },
+};
+
+const CONFIG_5_REPORT = {
+  code: 'LESMIS_enrolment_ece_5_target',
+  reportConfig: generateReport('er_district_ece_5_t', '0.86'),
+  frontEndConfig: { ...FRONT_END_CONFIG, name: 'Enrolment rate of 5 year old students in ECE' },
+};
+
+const addNewDashboardItemAndReport = async (
```

Comment:
This doesn't add any `dashboard_relation` entries in this migration, because these drill downs aren't directly attached to any dashboards, they're only accessed via drill down